src/rules/containers.ts

```diff
@@ -610,7 +610,7 @@ export const containerClusterInstanceCountInitialUint: YAMLRule = {
           pred: {
             And: {
               preds: [
-                {IsNotUint: {path: "cluster_instance_count.initial"}},
+                {NotUintString: {path: "cluster_instance_count.initial"}},
```

Comment:
With a theoretical standalone "IsNotTemplateFunc" predicate we could do this instead:
```suggestion
                {IsNotUint: {path: "cluster_instance_count.initial"}},
                {IsNotTemplateFunc: {path: "cluster_instance_count.initial"}},
```